test/wsdl-parse-test.js

```diff
@@ -60,4 +60,16 @@ describe(__filename, function() {
       done();
     });
   });
+
+  it('should parse types with same or no target namespaces', function(done) {
```

Comment:
is this test for parsing types with `same target namespaces` or `no target namespaces` or both?